pages/firststeps.md

```diff
@@ -25,41 +25,59 @@ A very large part of these steps is finding problems with these steps and this M
 There is no official deadline to complete these steps, but most candidates who are approved for the internship program finished the steps within 7-8 days. Good luck!
 
 
-##Step 0 - Introductory Steps:
+## Step 0 - Introductory Steps:
+
 1.Send us your resume again (along with a short description of yourself and what you would like to be called) to vi@ole.org.
-2. Watch [Open Learning Exchange](https://github.com/open-learning-exchange/open-learning-exchange.github.io) and Follow [Emily](https://github.com/EmilyLarkin), [Stuart](https://github.com/sente) and [Dogi](https://github.com/dogi) on GitHub. In GitHub we "Watch" organization repositories and "Follow" individuals. When you are on a repository or profile page, look for these buttons in the top right of the screen.  Also follow each other to see what others in the group are doing.
-3. We use Crowdin to translate our user interface in many different languages. If you know another language, go to [this Crowdin link](https://crowdin.com/project/open-learning-exchange/invite) to join Crowdin by creating an account. It would be very helpful for you to translate some of the words and familiarize yourself with the software. It would be great to get at least 10% on a language that you speak. If you want to spend additional time, feel free to provide translations to 100%.
 
-##Step 1 - BeLL Installation:
+2.Watch [Open Learning Exchange](https://github.com/open-learning-exchange/open-learning-exchange.github.io) and Follow [Emily](https://github.com/EmilyLarkin), [Stuart](https://github.com/sente) and [Dogi](https://github.com/dogi) on GitHub. In GitHub we "Watch" organization repositories and "Follow" individuals. When you are on a repository or profile page, look for these buttons in the top right of the screen.  Also follow each other to see what others in the group are doing.
```

Comment:
That is a very fine change, but you realize it is not related to the issue this pull request was addressing right?